notebooks/04-Anatomy_of_ASDF_Internals/04-Anatomy_of_ASDF_Internals.ipynb

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
<p>When we rebase this PR after merging #16, this file should disappear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>